nipype/utils/filemanip.py

```diff
@@ -736,7 +736,7 @@ def write_rst_header(header, level=0):
 
 def write_rst_list(items, prefix=""):
     out = []
-    for item in items:
+    for item in ensure_list(items):
         out.append("{} {}".format(prefix, str(item)))
```

Comment:
Where is this `ensure_list()` function defined?

Reply:
https://github.com/nipy/nipype/blob/4c48a185e4d190071f78a6766ca34c71ff61e10f/nipype/utils/filemanip.py#L523-L533